File: cmd/analyze/code-samples/split/java/sample-SPLIT.java
Visitor visitor = featureFlag.newVisitor("visitor_unique_id").build();
visitor.updateContext("isVip", true);
visitor.fetchFlags().whenComplete((instance, error) -> {
    
    Flag<String> btnColorFlag = visitor.getTreatment(key, "SPLIT-flag-java");

});

File: cmd/analyze/code-samples/split/php/sample-SPLIT.php

FeatureFlag::start("your_env_id", "your_api_key");
$visitor = FeatureFlag::newVisitor("your_visitor_id")->build();
$visitor->updateContext("isVip", true);
$visitor->fetchFlags();

$flag = $visitor->getTreatment($key, 'SPLIT-flag-php');

